ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.m

```diff
@@ -159,6 +159,14 @@ - (UIView *)miniAppViewWithName:(NSString *)name properties:(NSDictionary *)prop
     return rootView;
 }
 
+- (UIView *)miniAppViewWithName:(NSString *)name properties:(NSDictionary *_Nullable)properties
+            withSizeFlexibility:(NSInteger)sizeFlexibilty {
+    RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:self.bridge moduleName:name initialProperties:properties];
+    rootView.sizeFlexibility = (RCTRootViewSizeFlexibility)sizeFlexibilty;
+    rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
```

Comment:
Do we need to set the background color as well? Or only sizeFlexibility?

Reply:
Good question! By default I think it returns a black color view (need to test it). So I just maintained the same `RGB` values as other methods do.